### ISSUE TITLE
How to use the resnet for action recognition?

### ISSUE THREAD

**Issue by tingweiwang** [open-mmlab/mmaction#67]
Hi,
   We use the resnet (i.e. the resnet.py in "/mmaction/models/tenons/backbones") as the backbone for action recognition, and set pretrained='open-mmlab://resnet101_caffe'. However, a Top-1 accuracy of ~3% is achieved on UCF101 split 1.   
   Is the pretrained file resnet101_caffe-3ad79236.pth compatible with your resnet model defined in resnet.py?  The content of the configure file is as following.

model = dict(
    type='TSN2D',
    backbone=dict(
        type='ResNet',
        pretrained='open-mmlab://resnet101_caffe',
        depth=101,
        out_indices=[3],
        bn_eval=False,
        partial_bn=False),
    spatial_temporal_module=dict(
        type='SimpleSpatialModule',
        spatial_type='avg',
        spatial_size=7),
    segmental_consensus=dict(
        type='SimpleConsensus',
        consensus_type='avg'),
    cls_head=dict(
        type='ClsHead',
        with_avg_pool=False,
        temporal_feature_size=1,
        spatial_feature_size=1,
        dropout_ratio=0.8,
        in_channels=2048,
        init_std=0.001,
        num_classes=101))
train_cfg = None
test_cfg = None
# dataset settings
dataset_type = 'RawFramesDataset'
data_root = 'data/ucf101/rawframes'
img_norm_cfg = dict(
   mean=[104, 117, 128], std=[1, 1, 1], to_rgb=False)

data = dict(
    videos_per_gpu=24,
    workers_per_gpu=2,
    train=dict(
        type=dataset_type,
        ann_file='data/ucf101/ucf101_train_split_1_rawframes.txt',
        img_prefix=data_root,
        img_norm_cfg=img_norm_cfg,
        num_segments=3,
        new_length=1,
        new_step=1,
        random_shift=True,
        modality='RGB',
        image_tmpl='img_{:05d}.jpg',
        img_scale=256,
        input_size=224,
        div_255=False,
        flip_ratio=0.5,
        resize_keep_ratio=True,
        oversample=None,
        random_crop=False,
        more_fix_crop=False,
        multiscale_crop=True,
        scales=[1, 0.875, 0.75, 0.66],
        max_distort=1,
        test_mode=False),
    val=dict(
        type=dataset_type,
        ann_file='data/ucf101/ucf101_val_split_1_rawframes.txt',
        img_prefix=data_root,
        img_norm_cfg=img_norm_cfg,
        num_segments=3,
        new_length=1,
        new_step=1,
        random_shift=False,
        modality='RGB',
        image_tmpl='img_{:05d}.jpg',
        img_scale=256,
        input_size=224,
        div_255=False,
        flip_ratio=0,
        resize_keep_ratio=True,
        oversample=None,
        random_crop=False,
        more_fix_crop=False,
        multiscale_crop=False,
        test_mode=False),
    test=dict(
        type=dataset_type,
        ann_file='data/ucf101/ucf101_val_split_1_rawframes.txt',
        img_prefix=data_root,
        img_norm_cfg=img_norm_cfg,
        num_segments=25,
        new_length=1,
        new_step=1,
        random_shift=False,
        modality='RGB',
        image_tmpl='img_{:05d}.jpg',
        img_scale=256,
        input_size=224,
        div_255=False,
        flip_ratio=0,
        resize_keep_ratio=True,
        oversample='ten_crop',
        random_crop=False,
        more_fix_crop=False,
        multiscale_crop=False,
        test_mode=True))

optimizer = dict(type='SGD', lr=0.1, momentum=0.9, weight_decay=0.001)
optimizer_config = dict(grad_clip=dict(max_norm=40, norm_type=2))

lr_config = dict(
    policy='step',
    step=[30, 60])
checkpoint_config = dict(interval=1)

workflow = [('train', 1)]

log_config = dict(
    interval=20,
    hooks=[
        dict(type='TextLoggerHook'),
        # dict(type='TensorboardLoggerHook')
    ])

total_epochs = 80
dist_params = dict(backend='nccl')
log_level = 'INFO'
work_dir = './work_dirs/tsn_2d_rgb_resnet_seg_3_pretrained_g3'
load_from = None
resume_from = None

**Comment by zhaoyue-zephyrus** [comment]
Hi @tingweiwang 

The ResNet use `pytorch` style model as default but you are using the pretrained model with `caffe` style. Please consider use `pretrained='open-mmlab://resnet101'` instead and see if it works. I may upload a ResNet-50 model for reference as well.

**Comment by zhaoyue-zephyrus** [comment]
Hi @tingweiwang 

The TSN w/. ResNet-50 model has been uploaded in #86 . You can check it out and do minimal modification for ResNet-101.
